drop-in/src/main/java/com/adyen/checkout/dropin/internal/util/StoredUtils.kt

```diff
@@ -54,6 +54,17 @@ internal fun StoredPaymentMethod.mapStoredModel(
             )
         }
 
+        PaymentMethodTypes.PAY_BY_BANK_US -> {
+            GenericStoredModel(
+                id = id.orEmpty(),
+                imageId = type.orEmpty(),
+                isRemovable = isRemovingEnabled,
+                name = label.orEmpty(),
```

Comment:
Is this the only payment method where we will use the `label`?

Reply:
At this point yes, I talked to backend and we could move all the display logic in the future to backend and use this for all other payment methods.